tesseract/Dockerfile

```diff
@@ -0,0 +1,90 @@
+ARG ROSDISTRO=melodic
+
+FROM ros:$ROSDISTRO-perception
+MAINTAINER Levi Armstrong
+
+ARG ROSDISTRO
+ARG DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends \
+    ros-$ROSDISTRO-ompl \
+    python-catkin-tools \
+    ros-$ROSDISTRO-octomap-msgs \
+    ros-$ROSDISTRO-octomap \
+    libassimp-dev \
+    libtinyxml-dev \
+    tar \
+    wget \
+    ros-$ROSDISTRO-rviz \
+    ros-$ROSDISTRO-octomap-ros \
+    clang \
+    clang-tidy \
+    libomp-dev \
+    lcov \
```

Comment:
I updated the to follow the alphabetically ordered.

Reply:
>Additionally: not something you need to change, but the suggestion in #30 (comment) was not necessarily to apt install all packages in one RUN. Merely to use the suggested workflow / sequence of statements for all RUN lines which installed software using apt.

Are you recommending what is below?
```
RUN apt-get update
RUN apt-get install -y --no-install-recommends clang && apt-get clean && rm -rf /var/lib/apt/lists/*
RUN apt-get install -y --no-install-recommends clang-tidy && apt-get clean && rm -rf /var/lib/apt/lists/*
```

Reply:
No, it's ok to group `apt install`s of multiple packages. What I meant was: installing things right before they are used is beneficial as it can help Docker with reusing layers (the larger the set of packages you install, the higher the chance that you won't do that in any other `Dockerfile` with the exact same set -> no reuse of the layer).

So instead of installing all `.deb`s right at the beginning, you could install them in batches which make sense. I wouldn't go for separate lines for each package. That doesn't scale and is unmaintainable.

But again, not important here.